Description/Description.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Component\Resource\Description;
+
+use Puli\Repository\Api\Resource\PuliResource;
+
+/**
+ * Descriptive metadata for resources.
+ */
+class Description
+{
+    /**
+     * @var array
```

Comment:
I think we can limit them to be scalar values. If there is a use case for array values we could allow that later.